Chart.yaml

```diff
@@ -8,7 +8,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.29.0
+version: 1.30.1
```

Comment:
why is the version 1.30.1 and not 1.30.0?

Reply:
the latest release was 1.30.1 and I found that it was not correctly noted in the previous release. So I wanted to fix it in this one. Then, when the next gets created, which includes this postgres upgrade, it will be 1.31.0